index.js

```diff
@@ -1,6 +1,12 @@
 console.log("Hello World");
 
 const a = [1, 2, 10, 14, 5];
-a.forEach(n => console.log(n));
```

Comment:
Next time don't forget to properly indent your code. Let's follow coding style rules

Reply:
Will do. Thanks for the tip!